src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Debug/DebugProfileEnumValuesGeneratorTests.cs

```diff
@@ -32,8 +32,13 @@ public class DebugProfileEnumValuesGenerator_Tests
 
             Mock<ILaunchSettingsProvider> moqProfileProvider = new Mock<ILaunchSettingsProvider>();
             moqProfileProvider.Setup(p => p.CurrentSnapshot).Returns(testProfiles.Object);
+            var moqThreadingService = new Mock<IProjectThreadingService>();
```

Comment:
❕ Avoid using `Mock<T>` unless is it functionally impossible to create the equivalent code in the C# language. Instead, create a stub implementation of `IProjectThreadingService` which returns the expected value(s) and throws `NotImplementedException` elsewhere. This will allow features like finding all implementations to work correctly.